packages/lesswrong/components/ea-forum/AdvisorsRequestPage.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect, useState } from 'react';
+import { Components, registerComponent, getFragment } from '../../lib/vulcan-lib';
+import { useCurrentUser } from '../common/withUser';
+import { AnalyticsContext } from "../../lib/analyticsEvents";
+import { taglineSetting } from '../common/HeadTags';
+import { siteNameWithArticleSetting  } from '../../lib/instanceSettings';
+import AdvisorRequests from '../../lib/collections/advisorRequests/collection';
+
+const styles = (theme: ThemeType): JssStyles => ({
+  root: {
+    background: theme.palette.grey[0],
+    padding: '24px 32px',
+  },
+  headline: {
+    fontFamily: theme.typography.postStyle.fontFamily,
+    fontSize: 38,
+    fontWeight: 700
+  },
+  descriptionRow: {
+    marginTop: 20
+  },
+  link: {
+    color: theme.palette.primary.main
+  },
+  requestBtn: {
+    textTransform: 'none',
+    fontSize: 16,
+    boxShadow: 'none',
+    padding: '14px 24px',
+    marginTop: 30
+  },
+  requestBtnCancelText: {
+    color: theme.palette.grey[500],
+    marginTop: 10
+  },
+  advisorsHeadline: {
+    fontSize: 22,
+    fontWeight: 700,
+    paddingBottom: 10,
+    borderBottom: `2px solid ${theme.palette.primary.main}`,
+    marginTop: 30
+  }
+})
+
+
+// TODO: we decided not to implement this part for the beta test,
+//       so either finish or delete this after the test
+const AdvisorsRequestPage = ({classes}: {
```

Comment:
We originally intended to build a form on the forum, but later pivoted to just use Google Forms for the beta. Since we have a design for this page already, I left what we started building for now.